### ISSUE TITLE
Filter by type of entry (group, single chat, Gateway ID)

### ISSUE THREAD

**Issue by rugk** [threema-ch/threema-web#283]
See https://github.com/threema-ch/threema-web/issues/132#issuecomment-319662735

**Comment by dbrgn** [comment]
> As for implementation: Just a [filter icon](http://fontawesome.io/icon/filter/)…

Yes, but how should the dialog look? What options should there be? What types of input elements? How can the filter be reset? How do we avoid that people accidentally set a filter and then complain that they can't see some of the conversations?

**Comment by rugk** [comment]
Just some dropbox/checkbox thing, like that:
![grafik](https://user-images.githubusercontent.com/11966684/28900452-9666d2a6-77f1-11e7-8139-b9a2e97ccdec.png)

And, of course, the indicator icon should change (maybe a dot or the color) indicating that filtering is active.
